src/compiler/types.ts

```diff
@@ -5040,32 +5041,33 @@ namespace ts {
         ContainsTypeScript = 1 << 1,
         ContainsJsx = 1 << 2,
         ContainsESNext = 1 << 3,
-        ContainsES2017 = 1 << 4,
-        ContainsES2016 = 1 << 5,
-        ES2015 = 1 << 6,
-        ContainsES2015 = 1 << 7,
-        Generator = 1 << 8,
-        ContainsGenerator = 1 << 9,
-        DestructuringAssignment = 1 << 10,
-        ContainsDestructuringAssignment = 1 << 11,
+        ContainsES2019 = 1 << 4,
+        ContainsES2017 = 1 << 5,
+        ContainsES2016 = 1 << 6,
+        ES2015 = 1 << 7,
+        ContainsES2015 = 1 << 8,
+        Generator = 1 << 9,
+        ContainsGenerator = 1 << 10,
+        DestructuringAssignment = 1 << 11,
+        ContainsDestructuringAssignment = 1 << 12,
 
         // Markers
         // - Flags used to indicate that a subtree contains a specific transformation.
-        ContainsTypeScriptClassSyntax = 1 << 12,                // Decorators, Property Initializers, Parameter Property Initializers
-        ContainsLexicalThis = 1 << 13,
-        ContainsCapturedLexicalThis = 1 << 14,
-        ContainsLexicalThisInComputedPropertyName = 1 << 15,
-        ContainsDefaultValueAssignments = 1 << 16,
-        ContainsRestOrSpread = 1 << 17,
-        ContainsObjectRestOrSpread = 1 << 18,
-        ContainsComputedPropertyName = 1 << 19,
-        ContainsBlockScopedBinding = 1 << 20,
-        ContainsBindingPattern = 1 << 21,
-        ContainsYield = 1 << 22,
-        ContainsHoistedDeclarationOrCompletion = 1 << 23,
-        ContainsDynamicImport = 1 << 24,
-        Super = 1 << 25,
-        ContainsSuper = 1 << 26,
+        ContainsTypeScriptClassSyntax = 1 << 13,                // Decorators, Property Initializers, Parameter Property Initializers
+        ContainsLexicalThis = 1 << 14,
+        ContainsCapturedLexicalThis = 1 << 15,
+        ContainsLexicalThisInComputedPropertyName = 1 << 16,
+        ContainsDefaultValueAssignments = 1 << 17,
+        ContainsRestOrSpread = 1 << 18,
+        ContainsObjectRestOrSpread = 1 << 19,
+        ContainsComputedPropertyName = 1 << 20,
+        ContainsBlockScopedBinding = 1 << 21,
+        ContainsBindingPattern = 1 << 22,
+        ContainsYield = 1 << 23,
+        ContainsHoistedDeclarationOrCompletion = 1 << 24,
+        ContainsDynamicImport = 1 << 25,
+        Super = 1 << 26,
+        ContainsSuper = 1 << 27,

```

Comment:
It doesn't look like a good signal  :XD

Reply:
I just followed what #11407 did. https://github.com/Microsoft/TypeScript/pull/11407/files#diff-4b8bd1eea29904f1be39cd864e1a45c0